topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,468 @@
+v) The model is trained to classify images into categories (e.g., recognizing digits) and will train for 5 epochs.
+
+![Config.yaml file snapshot](../../images/galaxy-ludwig/explain_config.png "Config.yaml file snapshot."){: width="50%" height="5%"}
+
+
+# Galaxy-Ludwig Tool
+
+Ludwig simplifies the complexities of machine learning by automating essential steps such as data preprocessing, model architecture selection, hyperparameter tuning, and device management. To properly adapt the configuration for different datasets, users should consider the following parameters:
+
+- Input Features: Ensure the type reflects the nature of the data.
+- Encoder: Depending on the input type, this may need to be changed. For example, for image data, consider varying num_filters and filter_size. For text data, you might switch to type: rnn or type: transformer.
+- Output Features: Adjust the type according to the target variable. For instance, if it's a regression task, use type: numerical instead of category.
+- Trainer Parameters: Users can adjust epochs, batch_size, and learning_rate based on the dataset size, complexity, data behavior, and available computational resources.
+
+These are just a few of the many options that Ludwig provides out-of-the-box, allowing users to easily set and modify parameters. This is particularly beneficial for Galaxy users who are more interested in addressing their scientific questions than in navigating the intricacies of machine learning workflows.
+
+## Prepare environment and get the data 
+
+> <hands-on-title> Environment and Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial. If you are not inspired, you can name it *image classifier*.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the files from Zenodo or from the shared data library
+>
+>    ```
+>    https://zenodo.org/records/13292895/files/mnist_images.zip
+>    https://zenodo.org/records/13357260/files/mnist_dataset.csv 
+>    https://zenodo.org/records/13713095/files/config.yaml
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Check that the data format assigned for each file is **zip**, **csv**, and **yaml**, respectively.
+>    If it is not, follow the Changing the datatype tip.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add a tag (`Stacked CNN`) to the dataset corresponding to `config.yaml`
+>    This is important to trace back on what configurations the model was built on.
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+## Ludwig Experiment
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Ludwig Experiment](ludwig_experiment) %} with the following parameters:
```

Comment:
Adding tool's version would be nice! You can look at other ML tutorials such as deep learning or regression ones for example.

Reply:
Thank you. I just added.